### PR TITLE
LPS-161026 Validate creation comments block in Layout

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPS-161026

### CODE REVIEW

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:relevant

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectLayoutLocalServiceTest.java [hunk 1]
```diff
@@ -346,6 +347,178 @@ public void testAddObjectLayout() throws Exception {
 						"than 12"));
 		}
 
+		_objectDefinitionLocalService.deleteObjectDefinition(
```
Comment (seltong) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectLayoutLocalServiceTest.java:357]: @p-albuquerque, these tests need to be ordered by exception name and exception message. Could you please reorder these tests?

**Comment by seltong** [comment]
I will close this pull request and open a new one

**Comment by seltong** [comment]
Resent [here](https://github.com/liferay-objects/liferay-portal/pull/999).

**Pull request closed without merging**